### PR TITLE
Output update

### PR DESCRIPTION
**Pull request by yemca**

### CODE REVIEW

File: Analyzer.py [hunk 1]
```diff
@@ -161,21 +162,35 @@ def run_percentiles(data):
...
 
-def run_probability_dist(data, datatype="Interval"):
+def run_probability_dist(data, datatype):
     # change to histogram output
     # ordinal and interval
-    # TODO: make work for ordinal data
     if datatype == "Interval":
-        dist_data = data[-1] - data[0]
+        for column in data:
+            plt.figure()
```
Comment (ian-mcnichols) [inline Analyzer.py]: Can we add titles to the 1st 2 graphs? I'm not sure what they are

File: GUI.py [hunk 2]
```diff
@@ -136,7 +136,7 @@ def file_entry(self):
         self.fileName_layout = QHBoxLayout()
         self.fileName_layout.addWidget(self.fileName_lbl)
         self.fileName_layout.addWidget(self.fileName_txtbx)
-        self.submit_bttn = QPushButton("Submit")
+        self.submit_bttn = QPushButton("Load File")
```
Comment (ian-mcnichols) [inline GUI.py:144]: This crashes the GUI if the user doesn't select a file after opening the window

**Comment by laihipp** [comment]
- Titles on the graph
- Graphs don't currently save? Since there are so many, it might be a good idea to save them to their own folder
- Results window  - maybe have Results from Probability Distribution: See graphs?
- Move plotting functionality to visualizer and call from run_probabability_dist to keep current structure, is that possible?

**Comment by laihipp** [comment]
Runtime Error when testing:

/usr/local/bin/python3.9 "/Users/amberlaihipp/Documents/School/CS 499/CS_499_main/GUI.py"
4
4
operations:  ['Mean']
operations:  ['Mean', 'Median']
operations:  ['Mean', 'Median', 'Mode']
operations:  ['Mean', 'Median', 'Mode', 'Standard deviation']
operations:  ['Mean', 'Median', 'Mode', 'Standard deviation', 'Variance']
operations:  ['Mean', 'Median', 'Mode', 'Standard deviation', 'Variance', 'Percentiles']
operations:  ['Mean', 'Median', 'Mode', 'Standard deviation', 'Variance', 'Percentiles', 'Least square line']
operations:  ['Mean', 'Median', 'Mode', 'Standard deviation', 'Variance', 'Percentiles', 'Least square line', 'Probability distribution']
operations:  ['Mean', 'Median', 'Mode', 'Standard deviation', 'Variance', 'Percentiles', 'Least square line', 'Probability distribution', 'Spearman rank correction coefficient']
operations:  ['Mean', 'Median', 'Mode', 'Standard deviation', 'Variance', 'Percentiles', 'Least square line', 'Probability distribution', 'Spearman rank correction coefficient', 'Correlation coefficient']
running calculations!
running Mean
Results: [4.0, 4.0, 4.0, 4.0, 0.0]
running Median
Results: [4.0, 4.0, 4.0, 4.0, 0.0]
running Mode
Results: [4.0, 4.0, 4.0, 4.0, 0.0]
running Standard deviation
Results: [0.0, 0.0, 0.0, 0.0, 0.0]
running Variance
Results: [0.0, 0.0, 0.0, 0.0, 0.0]
running Percentiles
Results: [[[4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0], [4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0], [4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0], [4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0]], [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]]
running Least square line
Results: [nan, nan]
running Probability distribution
Results: (array([nan, nan, nan, nan]), 0.0, 0.0)
running Spearman rank correction coefficient
/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/scipy/stats/_stats_mstats_common.py:170: RuntimeWarning: invalid value encountered in double_scalars
  slope = ssxym / ssxm
/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/scipy/stats/_stats_mstats_common.py:187: RuntimeWarning: invalid value encountered in double_scalars
  slope_stderr = np.sqrt((1 - r**2) * ssym / ssxm / df)
/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/scipy/stats/_distn_infrastructure.py:1870: RuntimeWarning: invalid value encountered in true_divide
  x = np.asarray((x - loc)/scale, dtype=dtyp)
/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/scipy/stats/stats.py:4484: SpearmanRConstantInputWarning: An input array is constant; the correlation coefficient is not defined.
  warnings.warn(SpearmanRConstantInputWarning())
/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/numpy/lib/function_base.py:2691: RuntimeWarning: invalid value encountered in true_divide
  c /= stddev[:, None]
Results: [nan, nan]
running Correlation coefficient
Results: nan
plotting box plot
Plotting Histogram
Program Complete

Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)

**Review by laihipp — APPROVED** [review]

**Pull request merged**